### PR TITLE
Update Env and Docs

### LINKED ISSUE #88: Docker Docs Link
Docker updated their site and the link to the docker installation docs is now dead. (There is one in the README and in the Sphinx docs.)

Updated the link to http://docs.docker.com

### LINKED ISSUE #89: Fixtures are not included in the Manifest
The identifier types fixtures are not being included in the source distribution. The reason for this is that the path to the fixtures is not list in the manifest. To remedy, recursively include `name/fixtures` in Manifest.in.

### PR DESCRIPTION
**Pull request by damonkelley**
Fixes #89 and fixes #88 

Takes care of a few things:
- A more sane way of determining which tox envs to use locally
- Upgrade the dev env to use Django 1.8
- Fix the links to the docker installation docs
- The fixtures file is now included in the distribution.

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by ldko** [comment]
change README at ..."run the tests only with the development environment (i.e. with Django 1.7)" to Django 1.8

File: tests/settings/base.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
         'APP_DIRS': True,
         'OPTIONS': {
             'context_processors': [
+                'django.contrib.auth.context_processors.auth',
```
Comment (ldko) [inline tests/settings/base.py:45]: is this something to be added to docs for what needs to be added to settings?

Comment (damonkelley) [inline tests/settings/base.py:45]: yeah, probably

**Comment by somexpert** [comment]
:thumbsup:

File: tests/settings/base.py (hunk 1 above)
Comment (damonkelley) [inline tests/settings/base.py:45]: Updated in  4a66e8a

**Comment by ldko** [comment]
:+1:

**Pull request merged**